parsimonious/expressions.py

```diff
@@ -272,7 +273,7 @@ def _uncached_match(self, text, pos, cache, error):
     def _regex_flags_from_bits(self, bits):
         """Return the textual equivalent of numerically encoded regex flags."""
         flags = 'tilmsux'
-        return ''.join(flags[i] if (1 << i) & bits else '' for i in xrange(6))
+        return ''.join(flags[i] if (1 << i) & bits else '' for i in range(6))
```

Comment:
I'd prefer to keep the behavior unchanged here (rather than concretizing an array). I believe there's something from `six` we can import.